app/models/mixins/retirement_mixin.rb

```diff
@@ -51,7 +51,7 @@ def retires_on=(timestamp)
 
   def extend_retires_on(days, date = Time.zone.today)
     _log.info "Extending Retirement Date on #{self.class.name} id:<#{self.id}>, name:<#{self.name}> "
-    new_retires_date = date + days.to_i
+    new_retires_date = date + days.to_i.days
```

Comment:
@tinaafitz 
What is the date object coming into this function, would it have the time embedded in it? Since days returns a Duration object its possible that the time component could produce different results based on the timezone.